infra/tf/main.tf

```diff
@@ -135,20 +161,24 @@ resource "digitalocean_record" "ursa-dashboard-domain" {
 
 resource "digitalocean_droplet" "ursa-dashboard" {
   image      = var.droplet_image
-  name       = "dashboard"
+  name       = "ursa-dashboard"
   region     = var.droplet_region
-  size       = var.droplet_size
+  size       = "s-8vcpu-16gb-intel"
```

Comment:
Since there's no response, I'll regard that my proposed solution is acceptable
https://github.com/fleek-network/ursa/pull/447/commits/4377918ebed836e529d1d27ba53c9adcfb975481